apps/condo/domains/organization/integrations/sbbol/sync/syncTokens.js

```diff
@@ -5,13 +5,14 @@ const { getSbbolSecretStorage } = require('../utils')
  * @param {TokenSet} tokenInfoFromOAuth
  * @param {String} userId
  * @param {boolean} useExtendedConfig
+ * @param {String} organizationId
  * @return {Promise<void>}
  */
-const syncTokens = async (tokenInfoFromOAuth, userId, useExtendedConfig = false) => {
+const syncTokens = async (tokenInfoFromOAuth, userId, useExtendedConfig = false, organizationId) => {
```

Comment:
Typically, required function parameters come before optional ones
```suggestion
const syncTokens = async (tokenInfoFromOAuth, userId, organizationId, useExtendedConfig = false) => {
```